listener/subgraph_definition/dfusion

```diff
@@ -0,0 +1,22 @@
+specVersion: 0.0.1
+schema:
+    id: dfusion
+    file: 
+      /: schema.graphql
+dataSources:
+  - kind: ethereum/contract
+    name: SnappBaseEvents
+    source:
+      address: '0x9561C133DD8580860B6b7E504bC5Aa500f0f06a7'
+      abi: EnptyAbi.json
+    mapping:
+      kind: Mapping
+      apiVersion: 0.0.1
+      language: rust
+      entities: []
+      abis: []
+      eventHandlers:
+        - event: Deposit(uint16,uint8,uint128,uint256,uint16)
+          handler: rust
+      file: 
+        /: rustHandler.wasm
```

Comment:
Which part of our existing code base generates this file? Is this created during docker build? Will this file remain constant for ever onward?

Reply:
At this point this file is manually created. We will need to add a new (event, handler) item pair for every event we want to handle. 
Eventually, we could consider creating this manifest in code rather than reading it from a file but this way allowed us to reuse the most of the existing graph components.

I'd hope that at some point the graph might actually support external rust handlers in the core system.

Reply:
Perhaps, we could include something in the readme or wiki with instructions on this for the time being. If I remember correctly there is an outdated WIKI article with instructions on how to configure the listener to catch new events. [Here](https://github.com/gnosis/dex-services/wiki/Adding-Event-Listener)

Reply:
Good point. Added a section for the Graph (still with a few TODOs to be filled out throughout the course of this stack). Will eventually remove the Python tutorial once we remove that code.